content/lessons/chapter-7/outro-1.tsx

```diff
@@ -17,21 +17,18 @@ export default function Outro1({ lang }) {
 
   return (
     <ChapterEnd
-      className="top-[70px]"
+      className="bottom-[0px]"
```

Comment:
I think You can just remove this className, I will look to see why it exists in chapter 4